appveyor.yml

```diff
@@ -8,7 +8,7 @@ build_script:
 - call conda config --set always_yes yes --set changeps1 no
 - call conda info -a
 - call conda update -q conda
-- call conda install -c conda-forge python=3.7 re2c m2-bison xonsh llvmdev=11.1.0 jupyter xeus=1.0.1 xtl nlohmann_json cppzmq jupyter_kernel_test pytest
+- call conda install -c conda-forge python=3.7 re2c m2-bison xonsh llvmdev=11.1.0 jupyter xeus=1.0.1 xtl nlohmann_json cppzmq jupyter_kernel_test pytest symengine sympy
```

Comment:
I think we don't use appveyor anymore, so we can probably remove the whole file. In another PR.